functions.php

```diff
@@ -125,7 +125,7 @@ function twentysixteen_widgets_init() {
 	register_sidebar( array(
 		'name'          => esc_html__( 'Sidebar', 'twentysixteen' ),
 		'id'            => 'sidebar-1',
-		'description'   => '',
+		'description'   => __( 'Add widgets here to appear in your sidebar.', 'twentysixteen' ),
```

Comment:
Thanks for the pull request!

This looks good, but we need to escape it with [`esc_html`](https://developer.wordpress.org/reference/functions/esc_html/). Can you update your pull request? Thanks!